modules/deploy/partials/cloud/controlplane-api.adoc

```diff
@@ -191,7 +199,7 @@ To see the available regions for Redpanda Serverless, make a GET request to the
 
 [,bash]
 ----
-curl -H 'Authorization: Bearer <token>' https://api.redpanda.com/v1beta2/serverless/regions/
+curl -H 'Authorization: Bearer <token>' https://api.redpanda.com/v1beta2/serverless/regions?cloud_provider=CLOUD_PROVIDER_AWS
```

Comment:
Single quotes are needed around the last argument (the URL) because otherwise the question mark will be interpreted as a shell character instead of as part of the URL.